app/frontend/components/shared/form/email-form-control.tsx

```diff
@@ -29,23 +29,25 @@ interface IEmailFormControlProps extends FormControlProps {
   isRemovable?: boolean
   handleRemove?: () => void
   inputProps?: InputProps
+  inputRightElement?: JSX.Element
 }
 
 export const EmailFormControl = ({
   validate,
   label,
-  fieldName = "email",
+  fieldName,
```

Comment:
why remove the default?

Reply:
I needed to have a way to prevent this from registering - see the below conditional register